pkgs/development/python-modules/git-annex-adapter/default.nix

```diff
@@ -17,6 +17,10 @@ buildPythonPackage rec {
   };
 
   patches = [
+    (fetchpatch {
+      url = "${meta.homepage}/commit/6c210d828e8a57b12c716339ad1bf15c31cd4a55.patch";
```

Comment:
Please use the full url of the patch.

Reply:
Why?

Reply:
Because I can't click on it and the patch is not depending on the src we fetch which makes this easily bitrot.